doc/nrf/config_and_build/config_and_build_system.rst

```diff
@@ -169,22 +168,70 @@ Those additions are automatically included into the Zephyr build system using a
 
 You must be aware of these additions when you start writing your own applications based on this SDK.
 
-* The Kconfig option :kconfig:option:`CONFIG_WARN_EXPERIMENTAL` is enabled by default.
-  It gives warnings at CMake configure time if any experimental feature is enabled.
+.. _app_build_additions_experimental:
+
+Experimental option enabled by default
+======================================
+
+Unlike in Zephyr, the Kconfig option :kconfig:option:`CONFIG_WARN_EXPERIMENTAL` is enabled by default in the |NCS|.
+It gives warnings at CMake configure time if any :ref:`experimental <software_maturity>` feature is enabled.
+
+For example, when building a sample that enables :kconfig:option:`CONFIG_BT_EXT_ADV`, the following warning is printed at CMake configure time:
+
+.. code-block:: shell
+
+   warning: Experimental symbol BT_EXT_ADV is enabled.
+
+To disable these warnings, disable the :kconfig:option:`CONFIG_WARN_EXPERIMENTAL` Kconfig option.
+
+.. _app_build_additions_build_types:
+
+Custom build types
+==================
+
+A build type is a `CMake feature <CMAKE_BUILD_TYPE_>`_ that defines the way in which the configuration files are to be handled during the :ref:`configuration phase <configuration_system_overview_config>` to compile a build configuration for the application.
```

Comment:
Do we still use CMAKE_BUILD_TYPE anywhere?
We have originally used it and specified zdebug and zrelease build types but it was later abandoned and build type is controlled by providing sufixed prj.conf file.

Reply:
Rephrased.